ce/artifacts/artifact.ts

```diff
@@ -176,14 +176,12 @@ export class Artifact extends ArtifactBase {
         if (!installer) {
           fail(i`Unknown installer type ${installInfo!.installerKind}`);
         }
-
         await installer(this.session, activation, this.id, this.targetLocation, installInfo, events, options);
-
       }
 
       // after we unpack it, write out the installed manifest
       await this.writeManifest();
-
+      await this.loadActivationSettings(activation);
```

Comment:
Is this due to dynamically activating during activation?

Reply:
Yeah, kinda... in all that I somehow forgot to add the newly-installed artifact activation settings after install. 
If it was installed already, it worked, otherwise you'd have to activate again.